rholang/src/main/scala/coop/rchain/rholang/interpreter/RhoType.scala

```diff
@@ -41,4 +41,12 @@ object RhoType {
       }
   }
 
+  object Tuple2 {
+    def apply(b: Boolean) = Expr(GBool(b))
```

Comment:
I'm confused why the Tuple2 `apply` method takes a boolean and returns a Par that's a boolean. It should take a pair `(Par, Par)` and return a `Par`.

Reply:
You are right. This is a copy paste error.